agent/lib/kontena/weave_adapter.rb

```diff
@@ -0,0 +1,130 @@
+require_relative 'helpers/node_helper'
+
+module Kontena
+  class WeaveAdapter
+    include Helpers::NodeHelper
+
+    WEAVE_VERSION = 'git-bd19b8625908'
```

Comment:
We will change it to '1.1.0' when weave releases it.. after that it will change only when needed. One option is to override version through environment variable.